### ISSUE TITLE
`VERBOSITY` environment variable does not work as expected

### ISSUE THREAD

**Issue by iholder101** [kubevirt/kubevirt#10389]
**What happened**:
The `hack/manifests.sh` script uses the `VERBOSITY` environment and passes it as an argument to `./hack/build-manifests.sh`, which in turn creates manifests like Kubevirt CR.

However, setting this environment variable does not change the generated CR in any way.

**What you expected to happen**:
When doing:
```bash
> export VERBOSITY=10
> make cluster-up && make cluster-sync
> # let it finish...
> k get -n kubevirt kubevirt kubevirt -o yaml
```

I expect to see all components (e.g. virt-handler, virt-launcher, etc) defined with the defined verbosity.

**How to reproduce it (as minimally and precisely as possible)**:
Look at the previous point above.

**Environment**:
- KubeVirt version (use `virtctl version`): v1.0.0

**Comment by shubmjagtap** [comment]
@iholder101 can i work on this issue ? i need some time to get familiar with codebase and contrubuting guidelines .

**Comment by iholder101** [comment]
Yes indeed @shubmjagtap! Thanks for reaching out to help!
Take your time and please let me know if you need any assistance.

/assign @shubmjagtap

**Comment by shubmjagtap** [comment]
okay if  @Aryan8912 has attempted then let him work on this issue , i will unassign myself ( anyways im busy with other work )

**Comment by 7h3-3mp7y-m4n** [comment]
is the issue is still on ? so that I can work on it ...

**Comment by Aryan8912** [comment]
Ya work on it, I am busy other work

**Comment by 7h3-3mp7y-m4n** [comment]
okay

**Comment by 7h3-3mp7y-m4n** [comment]
@iholder101  yes maybe you are right , Iet me apply it and see the outcomes . Maybe it will resolve the conflicts .

**Comment by alicefr** [comment]
This is being assigned for too long without updates so I'm moving it back to unassigned. If you are still working on this, please assign to yourself again so we can avoid duplicated work

/unassign
